docker/docker-compose.yaml

```diff
@@ -69,6 +69,19 @@ services:
     volumes:
       - jenkins_shared:/shared
 
+  jenkins-agent-ruby:
```

Comment:
one minor thing left: update `JENKINS_AGENT_NAMES` in this file (line 19) to include `agent-ruby`